setup.py

```diff
@@ -8,7 +7,7 @@
     history = f.read()
 
 setuptools.setup(name='Flask-XStatic',
-                 version='0.0.1+dev',
+                 version='0.0.2+dev',
```

Comment:
Can you remove that hunk? While the next version is not released it's still `0.0.1+dev`. Thanks!